sources/packages/backend/apps/queue-consumers/src/processors/schedulers/cas-integration/cas-supplier-integration.scheduler.ts

```diff
@@ -63,19 +62,16 @@ export class CASSupplierIntegrationScheduler extends BaseScheduler<void> {
           `Records updated: ${suppliersUpdated}.`,
         ],
         processSummary,
+        { throwOnError: true },
       );
     } catch (error: unknown) {
+      // Throw an error to force the queue to retry.
       if (error instanceof CustomNamedError) {
-        if (error.name === CAS_AUTH_ERROR) {
-          // Throw an error to force the queue to retry.
-          throw new Error(
-            `Unable to request data to CAS due to an authentication error.`,
-          );
-        }
-        const errorMessage = "Unexpected error while executing the job.";
-        processSummary.error(errorMessage, error);
-        return [errorMessage];
+        throw new Error(error.message);
```

Comment:
We won't throw the cause here? `cause: error`

Reply:
For custom-named errors, we assume the message will be friendly.